plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/controller/CodeTransformChatController.kt

```diff
@@ -680,6 +681,7 @@ class CodeTransformChatController(
         )
 
     private suspend fun handleCodeTransformResult(result: CodeModernizerJobCompletedResult) {
+        LOG.info { "CodeModernizerJobCompletedResult: $result"}
```

Comment:
Just confirming we do want these logs and not just for debugging right?

Reply:
These logs will help us debug better yes